src/main/java/com/groupesan/project/java/scrumsimulator/mainpackage/ui/panels/SolutionListPane.java

```diff
@@ -20,24 +20,24 @@
 public class SolutionListPane extends JFrame implements BaseComponent {
     private Player player;
     private String simulationId;
+    private List<SolutionWidget> widgets = new ArrayList<>();
+    private JPanel subPanel = new JPanel();
 
     public SolutionListPane(Player player, String simulationId) {
         this.player = player;
         this.simulationId = simulationId;
         this.init();
     }
-    private List<SolutionWidget> widgets = new ArrayList<>();
-    private JPanel subPanel = new JPanel();
 
-    public void init(){
+    public void init() {
         setDefaultCloseOperation(JFrame.DISPOSE_ON_CLOSE);
         setTitle("List of Solutions");
         setSize(500, 300);
 
-        GridBagLayout myGridbagLayout = new GridBagLayout();
-        JPanel myJpanel = new JPanel();
-        myJpanel.setBorder(new EmptyBorder(10, 10, 10, 10));
-        myJpanel.setLayout(myGridbagLayout);
+        GridBagLayout layout = new GridBagLayout();
+        JPanel mainPanel = new JPanel();
+        mainPanel.setBorder(new EmptyBorder(10, 10, 10, 10));
+        mainPanel.setLayout(layout);
```

Comment:
@yeain-shukla Can you please elaborate, why they are changes in this file SolutionListPane.java, which are not necessary to make changes related to your task, I see just the variable names have been changed and few lines have unnecessary updates, please retain only changes needed for task and update the PR accordingly.
Thank you.